docs/extensions.rst

```diff
@@ -145,10 +370,10 @@ function may be implemented:
         parser.add_argument('--with-awesome',
                             dest='extensions',
                             action='append_const',
-                            const=extend_pyscaffold,
+                            const=extend_scaffold,
                             help='generate awesome extra files')
 
-Note that, in this case, an option with the ``append_const`` action is created,
+In this case, an option with the ``append_const`` action is created,
 with ``extensions`` as ``dest`` and the extension function as ``const``.
```

Comment:
If I got it right, the `extensions` destination is our special convention here that holds all activated extensions and the PyScaffold code will use the `extensions` attribute from the argparse namespace to look them up. If this is right maybe it makes sense to clarify this with a sentence or two at that point.